server/database/build.js

```diff
@@ -0,0 +1,14 @@
+const { readFileSync } = require('fs');
+const path = require('path');
+const connect = require('./connection.js');
+
+const error = new TypeError('error in building database');
+const sql = readFileSync(path.join(__dirname, 'build.sql')).toString();
+
+const DBRun = () => connect.query(sql);
+DBRun().then((result) => {
+  if (result) {
+    return result;
```

Comment:
I don't understand the point of this line. I would prefer to log a message to the dev.